### PR TITLE
Fix pixelated logo in tweets

### PR DESCRIPTION
**Pull request by ercohen14**
This PR will fix issue #150

### CODE REVIEW

**Comment by ercohen14** [comment]
I had trouble figuring out how to test this PR, is there a way to test a twitter/linkedin image via local host? Let me know @etagwerker if this is the solution you are looking for.

File: _layouts/default.html [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
     <!-- Twitter summary card with large image must be at least 280x150px -->
-    <!-- meta name="twitter:image:src" content="http://www.example.com/image.html" -->
+    <meta name="twitter:image:src" content="{{site.domain_name}}/assets/profile.png">
 
     <!-- Open Graph data -->
     <meta property="og:title" content="{{ page_title }}" />
     <meta property="og:type" content="article" />
     <meta property="og:url" content="{{site.domain_name}}{{site.baseurl}}{{ page.url }}" />
-    <meta property="og:image" content="{{site.domain_name}}/assets/isotype.png" />
+    <meta property="og:image" content="{{site.domain_name}}/assets/profile.png" />
```
Comment (mauro-oto) [inline _layouts/default.html]: @ercohen14 I think this is trying to use the `ombulabs.com` domain only, it should be:

`{{site.baseurl }}/assets/images/profile.png`

in order to reach https://www.ombulabs.com/blog/assets/images/profile.png

And same for the meta tag above.

**Comment by etagwerker** [comment]
@ercohen14 One simple way to test this would be to deploy it to a staging server and then use https://cards-dev.twitter.com/validator with the staging url.

**Comment by ercohen14** [comment]
@etagwerker What staging server do you recommend? Should I make an account in Heroku and put it there?

**Comment by mauro-oto** [comment]
@ercohen14 we already have a staging server here: http://ombulabs-staging.herokuapp.com/

I just sent you an invite to the Heroku dashboard for it: https://dashboard.heroku.com/apps/ombulabs-staging

Thanks for updating the image URL! I think you also need to update the path in line 32, since it seems to be the same image and the path is still incorrect in that one.

**Comment by ercohen14** [comment]
Ok, just fixed it. How can I tell what PRs have gone into staging? It doesn't look updated to me. 

Either way, it seems that the card is not working. 

screen shot 2018-08-28 at 11 37 24 am

**Comment by ercohen14** [comment]
The card validator tests are working now: 

screen shot 2018-09-10 at 3 29 52 pm

**Pull request merged**